apps/spruce/src/pages/version/TestAnalysis/GroupedTestMapList/index.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from "@emotion/styled";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { TaskBuildVariantField } from "../types";
+import FailedTestGroup from "./FailedTestGroup";
+
+interface GroupedTestMapListProps {
+  groupedTestsMapEntries: [string, TaskBuildVariantField[]][];
+}
+const GroupedTestMapList: React.FC<GroupedTestMapListProps> = ({
+  groupedTestsMapEntries,
+}) => (
+  <>
+    {groupedTestsMapEntries.map(([testName, tasks]) => (
+      <SpacedDiv>
+        <FailedTestGroup key={testName} tasks={tasks} testName={testName} />
+      </SpacedDiv>
+    ))}
+  </>
```

Comment:
Could we use a wrapper container with `gap` (instead of putting `margin`s on each element)?
```suggestion
  <Container>
    {groupedTestsMapEntries.map(([testName, tasks]) => (
      <FailedTestGroup key={testName} tasks={tasks} testName={testName} />
    ))}
  </Container>
);

const Container = styled.div`
  display: flex;
  flex-direction: column;
  gap: ${size.s};
`;
```